controllers/chat/moveToAnon.js

```diff
@@ -30,6 +31,11 @@ const moveToAnon = async (req, res) => {
     user_id: targetUserId
   });
 
+  const [emoji, color] = await Promise.all([
+    BetterSocialConstantListUtils.getRandomEmoji(),
+    BetterSocialConstantListUtils.getRandomColor()
+  ]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `Promise.all` here is a good practice as it allows both promises to run concurrently, which can improve performance. However, make sure that both functions `getRandomEmoji` and `getRandomColor` are independent and do not rely on each other's results.
```diff
+   const [emoji, color] = await Promise.all([
+     BetterSocialConstantListUtils.getRandomEmoji(),
+     BetterSocialConstantListUtils.getRandomColor()
+   ]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->